packages/@react-spectrum/picker/docs/Picker.mdx

```diff
@@ -0,0 +1,349 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/picker';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Picker, Item, Section} from '@react-spectrum/picker';
+import Book from '@spectrum-icons/workflow/Book';
+import BulkEditUsers from '@spectrum-icons/workflow/BulkEditUsers';
+import Draw from '@spectrum-icons/workflow/Draw';
+import ColorPalette from '@spectrum-icons/workflow/ColorPalette';
+import AlignLeft from '@spectrum-icons/workflow/AlignLeft';
+import {Text} from '@react-spectrum/typography';
+```
+
+---
+category: Pickers
+---
+
+# Picker
+
+<p>{docs.exports.Picker.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Picker', 'Item', 'Section']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/dropdown/'}
+  ]} />
+
+## Example
+```tsx example
+<Picker label="Choose frequency" onSelectionChange={selected => console.log(selected)}>
+  <Item uniqueKey="rarely">Rarely</Item>
+  <Item uniqueKey="sometimes">Sometimes</Item>
+  <Item uniqueKey="always">Always</Item>
+</Picker>
+```
+
+## Content
+Picker follows the [Collection Components](../react-stately/collections.html) API, accepting both static and dynamic forms of children. Similar to the Menu, Picker accepts an `<Item>` component, each with a `uniqueKey` prop. Basic usage of the Picker, seen in the example above, shows multiple options populated with a string. This implementation would be used when the list of options is finite and known.
+
+The dynamic method (shown below) would be better suited to use if the options came from a data object such as values returned from an API call. Providing the data in this way allows for Picker to automatically cache the rendering of each item, which dramatically improves performance.
+
+As seen below, an iterable list of options is passed to the Picker using the `items` prop. The value to be passed to the `onSelectionChange` handler can be set via the `uniqueKey` prop. Alternatively, if the options dataset contains an `id` key, as shown in the example below, this can be used instead. See the <a href="#events" title="Events">events</a> section for more detail on selection.
```

Comment:
> The value to be passed to the `onSelectionChange` handler can be set via the `uniqueKey` prop.

This is a bit confusing. I think I'd say something like "Each item accepts a `uniqueKey` prop, which is passed to the onSelectionChange handler to identify the selected item."

Reply:
"Alternatively, if the item objects contain an `id` property, as shown in the example below, then this is used automatically and a `uniqueKey` prop is not required.